frontend/src/app/app.component.html

```diff
@@ -8,14 +8,29 @@
     <app-nav-bar-menu></app-nav-bar-menu>
   </mat-drawer>
   <mat-drawer-content class="overall-wrapper">
-    <div class="navBar">
+    <div
+      class="navBar"
+      *ngIf="(fullscreenService.isFullscreen$ | async) === false"
```

Comment:
Isn't this better than explicitly checking for `false`?

```suggestion
      *ngIf="!(fullscreenService.isFullscreen$ | async)"
```

Reply:
Yes in theory I would agree with you (and I did it like this before) but eslint had something against using `!` together with an async pipe. Didn't look up the actual reason behind this but now I was curious and [here](http://codelyzer.com/rules/template-no-negated-async/) is the reason behind that.